01-syntax/05-oop/Sequence.js

```diff
@@ -1 +1,52 @@
-// → Your code here.
+function Interface(arr) {
```

Comment:
Too much code  for that kind on task 

I want only 'Sequence' in that file, also use 

This one

https://github.com/nesterone/sf-eng-init-gl-spring-2016/blob/master/01-syntax/05-oop/Sequence.js#L1